src/descend.ml

```diff
@@ -1470,15 +1470,16 @@ value print conf base p =
   let templ =
     match p_getenv conf.env "t" with
     [ Some ("F" | "L" | "M") -> "deslist"
+    | Some ("H" | "I" | "A") -> "destable"
     | Some _ -> ""
     | _ -> "desmenu" ]
   in
   if templ <> "" then Perso.interp_templ templ conf base p
   else
     match (p_getenv conf.env "t", p_getint conf.env "v") with
-    [ (Some "A", Some v) -> print_aboville conf base v p
```

Comment:
It goes with what is above A and H are in destable template file, we kept same name, it's retrocompatible, we only add two new names B/K if someone want to see the “old” render from those pages from Ocaml.

20:20 <@A2`> et donc d'Aboville et destable sont maintenant dans un fichier de template, mais on les a gardé dans l'Ocaml, avec d'ailleurs son container-fluid (mais c'est  pu accessible sans triturer un peu le template les t=B ou t=K) > enfin je sais pas pourquoi j'explique tu le comprends à le lire :D

> ou pas…